controllers/machinehealthcheck_controller_test.go

```diff
@@ -187,10 +201,14 @@ var _ = Describe("MachineHealthCheck", func() {
 					},
 				},
 			}
-			mhc.Default()
+			// removed because to fix "when a Machine has no Node ref for less than the NodeStartupTimeout" issue
+			//mhc.Default()
 		})
 
+		// delete all the nodes and mhc in the cluster, if the dekete get issues, it does not step
```

Comment:
```suggestion
		// Delete all the nodes and mhc in the cluster. If the delete has issues, it does not step
```

Is "step" the right word here or did you mean "stop" or something else?